save/src/protos.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package protos;
+
+message Meta {
+    // Number of voxels along the edge of a chunk
+    uint32 chunk_size = 1;
+}
+
+message Node {
+    // Entities whose origins lie within this node
+    repeated Archetype archetypes = 1;
+
+    // Voxel data for each modified chunk
+    repeated Chunk chunks = 2;
+}
+
+// A set of entities, all of which have the same components
+message Archetype {
+    // Entity IDs
+    repeated fixed64 entities = 1;
+
+    // Type of components stored in each column
+    repeated ComponentType component_types = 2;
+
+    // Columns of component data
+    repeated bytes component_data = 3;
+}
+
+message Chunk {
+    // Which dodecahedron vertex is associated with this chunk
+    uint32 vertex = 1;
```

Comment:
Protos don't have smaller integer types (see https://protobuf.dev/programming-guides/proto3/#scalar); even enums are just a thin cosmetic wrapper over `int32`. Instead, the variable-length "varint" encoding is used for all non-`fixed` integers, which will in practice always represent such small values with a single byte.

A previous draft defined vertex/side enums mirroring those of dodeca.rs, but I dropped those for two reasons:
- `prost` still ultimately works in integer terms so that unknown values can be represented, since enums in protos can generally expand in the future.
- `protoc` doesn't allow enums in the same scope to reuse the same name, so we couldn't have both `Vertex::A` and `Side::A`.